MSTuringEmbeddingOnlyScenario.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
+namespace VectorIndexScenarioSuite
+{ 
+    internal class MSTuringEmbeddingOnlyScenario : BigANNBinaryEmbeddingOnlyScearioBase
+    {
+        protected override string BaseDataFile => "base";
+        protected override string BinaryFileExt => "fbin";
+        protected override string QueryFile => "query";
+        protected override string GetGroundTruthFileName => "ground_truth";
+        protected override string PartitionKeyPath => "/id";
+        protected override string EmbeddingColumn => "embedding";
+        protected override string EmbeddingPath => $"/{EmbeddingColumn}";
+        protected override VectorDataType EmbeddingDataType => VectorDataType.Float32;
+        protected override DistanceFunction EmbeddingDistanceFunction => DistanceFunction.Euclidean;
+        protected override ulong EmbeddingDimensions => 100;
+        protected override int MaxPhysicalPartitionCount => 56;
+        protected override string RunName => "msturing-embeddingonly-" + guid;
+
+        public MSTuringEmbeddingOnlyScenario(IConfiguration configurations) : 
+            base(configurations, ComputeInitialAndFinalThroughput(configurations).Item1)
+        {
+        }
+
+        public override void Setup()
+        {
+            this.CosmosContainer.ReplaceThroughputAsync(ComputeInitialAndFinalThroughput(this.Configurations).Item2).Wait();
+        }
+
+        private static (int, int) ComputeInitialAndFinalThroughput(IConfiguration configurations)
+        {
+            // seting the throughput for the container initially for creation and then bumping it up
+            int init_RU = Convert.ToInt32(configurations["AppSettings:cosmosContainerRUInitial"]);

```

Comment:
Can we have some good defaults here for initial / final RU and only use the config for overriding default values. 
The goal with scenarios is for customers to use this app and see great perf out of the box without having to have play with correct RU number to use. So defaults are important.